include/ck/tensor_operation/gpu/device/masking_specialization.hpp

```diff
@@ -47,13 +50,39 @@ struct MaskOutUpperTrianglePredicate
         return operator()(m + m_tile - 1, n);
     }
 };
+struct MaskUpperTriangleFromBottomRightPredicate
+{
+    MaskUpperTriangleFromBottomRightPredicate() : offset_(0) {}
+    __host__ __device__ void SetOffset(const index_t offset) { offset_ = offset; }
+    __host__ __device__ constexpr bool operator()(index_t m, index_t n) const
+    {
+        return n > (m + offset_);
+    }
+
+    __host__ __device__ constexpr bool
+    IsTileSkippable(index_t m, index_t n, index_t m_tile, index_t /*n_tile*/) const
```

Comment:
Rename `m` to `m_tile_orig`, `n` to `n_tile_orig`, `m_tile` to `m_tile_size`, `n_tile` to `n_tile_size`

Reply:
done